i18n/kr.yaml

```diff
@@ -0,0 +1,6 @@
+integrations_and_platforms:
```

Comment:
Also this file. Can we use the ISO 639-1 code to refer to Korean language?

It should be `ko`, imho :)